<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>1066</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>APT Data Input</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Data File Name:</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_data_fname">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>481</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>test_simple_syn.pos</string>
     </property>
     <property name="placeholderText">
      <string>pos or txt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Range File Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_rrng_fname">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>481</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>test_simple_syn.rrng</string>
     </property>
     <property name="placeholderText">
      <string>rrng</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_load_data">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>140</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Load data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_data_fname">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_range_fname">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>90</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_show_ion_map">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>140</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Show ion map</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_point_size">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ion map point size (pixel):</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>741</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Cluster Analysis</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ion Types:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_ion_types">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>611</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>all</string>
     </property>
     <property name="placeholderText">
      <string>Ranged, All, Noise, or A1B1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Method:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_method">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OPTICS-APT</string>
     </property>
     <property name="placeholderText">
      <string>DBSCAN or OPTICS-APT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Max search distance (eps):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_eps">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>10.0</string>
     </property>
     <property name="placeholderText">
      <string>0-inf</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>110</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Minimum point for core (minpts):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_minpts">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>110</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>10</string>
     </property>
     <property name="placeholderText">
      <string>2-inf</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>711</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Experimental</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Material Density (#/nm3):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_density">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>88.48</string>
      </property>
      <property name="placeholderText">
       <string>88.48</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Detector Efficiency:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_det_eff">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0.37</string>
      </property>
      <property name="placeholderText">
       <string>0.37</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Solute Concentration:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_solute_con">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>20</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0.14</string>
      </property>
      <property name="placeholderText">
       <string>0.14</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_expert">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>711</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Expert</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Local maxima neighbors k:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_k">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>20</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
      <property name="placeholderText">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Significant separation (0.0-1.0):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_significant_separation">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0.75</string>
      </property>
      <property name="placeholderText">
       <string>0.75</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>60</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Node Similarity:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Cluster membership probability threshold (0.0-1.0):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_cluster_member_prob">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>59</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0.5</string>
      </property>
      <property name="placeholderText">
       <string>0.5</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_similarity">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>60</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0.9</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Min node size:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_min_node_size">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="button_run">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_show_bg">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>160</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>show background</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_terminate">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Terminate</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_show_expert">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Show expert options</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_show_visual">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>160</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Show Visual</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>711</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Post-processing</string>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Min cluster size:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_min_cluster_size">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>None</string>
      </property>
      <property name="placeholderText">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Estimated Background RD:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_est_bg">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>820</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>730</y>
      <width>741</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Output Filename:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_output_fname">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>test_optics</string>
     </property>
     <property name="placeholderText">
      <string>Filename base, no extension</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_save_output">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>40</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_clear_status">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>990</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>850</y>
      <width>731</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main</string>
    </property>
   </widget>
   <addaction name="menuMain"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_clear_status</sender>
   <signal>clicked()</signal>
   <receiver>textBrowser</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>969</x>
     <y>713</y>
    </hint>
    <hint type="destinationlabel">
     <x>970</x>
     <y>630</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
